portfolio/src/main/webapp/index.html

```diff
@@ -1,83 +1,99 @@
+      During my first two years of college, I have taken technical courses 
+      in databases, data structures and algorithms, 
+      fundamentals of computer systems, 
+      introduction to electrical engineering, 
+      introductory Python and Java. 
+      In addition to speaking English and Chinese fluently, 
+      I am also learning Spanish.
+      </p>
+      <p>
+      While on campus, I am involved with Global Youth Mentorship Initiative 
+      (GYMI), Matriculate, Women in Computer Science (WISC), 
+      Intramural Program, and three on-campus jobs.
+      </p>
 
-		<h4>Background</h4>
-		<p>
-			I am a first-generation immigrant from Fuzhou, China and 
-			a first-generation college student. My family and I
-			lived in Atlanta, GA, USA, where I received
-			my high school diploma from Cross Keys High School. 
-			In high school, I was involved with Vex Robotics,
-			Technology Students Association (TSA),
-			Girls' Varsity Basketball, 
-			Mu Alpha Theta National Mathematics Honors Society, 
-			National Honors Society, and Beta Club.
-		</p>
-		<div class="skill">
-			<h4>Skill/Interest</h4>
-			<ul>
-				<li>Programming: Java, Python (Numpy, Pandas, Matplotlib, Flask),
-					C, C++, HTML, CSS</li>
-				<li>Tool: Linux, MacOS, Windows, Photoshop, Autodesk Fusion, Git</li>
-				<li>Language: Fluent Mandarin, beginner Spanish</li>
-				<li>Hobbies: Exercise, run, photography, Chinese calligraphy</li>
-			</ul>
-		</div>
-	</div>
-</body>
+      <h4>Background</h4>
+      <p>
+      I am a first-generation immigrant from Fuzhou, China and 
+      a first-generation college student. My family and I
+      lived in Atlanta, GA, USA, where I received
+      my high school diploma from Cross Keys High School. 
+      In high school, I was involved with Vex Robotics,
+      Technology Students Association (TSA),
+      Girls' Varsity Basketball, 
+      Mu Alpha Theta National Mathematics Honors Society, 
+      National Honors Society, and Beta Club.
+      </p>
+      <div class="skill">
+        <h4>Skill/Interest</h4>
+        <ul>
+          <li>Programming: Java, Python (Numpy, Pandas, Matplotlib, Flask),
+            C, C++, HTML, CSS</li>
+          <li>Tool: Linux, MacOS, Windows, Photoshop, Autodesk Fusion, Git</li>
+          <li>Language: Fluent Mandarin, beginner Spanish</li>
+          <li>Hobbies: Exercise, run, photography, Chinese calligraphy</li>
+        </ul>
+      </div>
+      <div class="comment">
+        <h4>Comments: </h4>
+        <ul id="comment-container"></ul>
+        <form action='/data' method='POST' class="comment-form">
+          <p>Leave your comment below: </p>
+          <Label for="name">Your name: </label>
+          <input name="name" type="text">
+          <br><br>
+          <label for="comment"> Comment: </label>
+          <textarea name="comment"></textarea></br>
+          <input type="submit">
+        </form>
+      </div>
```

Comment:
This is the new portion of this file right? It looks fine but I'd remove the space after `Comments:`. There's going to be a line break after it anyway so is slightly untidy.

Reply:
This is an added section to display the stored comments and as well as a form for others to leave comments. I will remove the trailing white spaces.